### PR TITLE
Add new plaid-link package

### PR DESCRIPTION
**Pull request by pbernasconi**
This PR adds definition for [Plaid Link](https://plaid.com/docs/quickstart), as `plaid-link`, a frontend drop-in module for interacting with the [Plaid API](https://plaid.com). It will often be consumed by Angular/Ionic/any frontend project written in Typescript.

More detailed parameter/docs reference can be found at: https://plaid.com/docs/api

- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `tsc` without errors.
- [X] Run `npm run lint package-name` if a `tslint.json` is present.

### CODE REVIEW

File: plaid-link/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": false,
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline plaid-link/tsconfig.json]: new packages should have `strictNullchecks` `true`.

File: plaid-link/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// Type definitions for Plaid Link
```
Comment (mhegazy) [inline plaid-link/index.d.ts]: no version?

Comment (mhegazy) [inline plaid-link/index.d.ts]: Is this package available on npm? if so what is the package name?

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by paulvanbrenk** [comment]
Closing this PR, as it's gotten pretty stale. Please ping us to re-open if you made the requested changes, or start a new PR.

**Pull request closed without merging**